test/tmisc.c

```diff
@@ -3851,20 +3851,10 @@ test_misc20(void)
         return;
     }
 
-    /* Verify that chunks with dimensions that are too large get rejected */
-
     /* Create a dataset creation property list */
     dcpl = H5Pcreate(H5P_DATASET_CREATE);
     CHECK(dcpl, FAIL, "H5Pcreate");
 
-    /* Try to use chunked storage for this dataset */
```

Comment:
This is now allowed. We could cap the high bound to test that it's rejected in that case but that's already tested elsewhere.